File: Daily Test/C/Inverted Right Angles Triangle Pattern 18.8.2025.c
#include <stdio.h>
int main() {
    int N; 
    scanf("%d", &N);
    int cnt = 1 + 4 * (N - 1);

    for (int i = 1; i <= cnt; i++) {
        if (i % 2 == 1) printf("*");
        else printf("-");
    }
    cnt -= 4;
    printf("\n");

    for (int i = 1; i < N; i++) {
        for (int k = 0; k < i * 2; k++) printf("-");
        for (int j = 1; j <= cnt; j++) {
            if (j == 1 || j == cnt / 2 + 1 || j == cnt) printf("*");
            else printf("-");
        }
        cnt -= 4;
        printf("\n");
    }
    return 0;
}

File: Daily Test/Python/Inverted Right Angles Triangle Pattern 18.8.2025.py
N = int(input())
cnt = 1
cnt += 4 * (N - 1)
for i in range(1, cnt + 1):
    if i % 2 == 1:
        print('*', end='')
    else:
        print('-', end='')
cnt -= 4
print()
for i in range(1, N):
    print(i * 2 * '-', end="")
    for j in range(1, cnt + 1):
        if j == 1 or j == cnt // 2 + 1 or j == cnt:
            print('*', end='')
        else:
            print('-', end='')
    cnt -= 4
    print()